src/commands/account/add.ts

```diff
@@ -0,0 +1,194 @@
+import axios from "axios";
+import chalk from "chalk";
+import fs from "fs-extra";
+import retry from "async-retry";
+import { prompt } from "inquirer";
+import Command from "../../base";
+import { flags } from "@oclif/command";
+import promptEmail from "email-prompt-ts";
+import eraseLines from "../../utils/erase-lines";
+import { createDebugLogger } from "../../utils/output";
+import { validate as validateEmail } from "email-validator";
+import { GLOBAL_CONF_PATH, REGIONS_API_URL } from "../../constants";
+
+interface IAccount {
+  email: string;
+  api_token: string;
+  region: string;
+}
+
+interface IAccounts {
+  [key: string]: IAccount;
+}
+
+interface ILiaraJson {
+  api_token?: string;
+  region?: string;
+  current?: string;
+  accounts?: IAccounts;
+}
+
+export default class AccountAdd extends Command {
+  static description = "add an account";
+
+  static flags = {
+    ...Command.flags,
+    name: flags.string({ char: "n", description: "account name" }),
+    email: flags.string({ char: "e", description: "your email" }),
+    password: flags.string({ char: "p", description: "your password" }),
+  };
+
+  async run() {
+    const { flags } = this.parse(AccountAdd);
+    const debug = createDebugLogger(flags.debug);
+    const liara_json: ILiaraJson = this.gatherLiaraJson();
```

Comment:
Please rename `gatherLiaraJson` to `readGlobalLiaraJson`. We might mistaken this with the project's local `liara.json`.